### ISSUE TITLE
19.07.3 regression: installing an app doesn't show in the menu unless logging out/in again

### ISSUE THREAD

**Issue by jonglezb** [openwrt/luci#4077]
In 19.07.2, after installing a LuCI app, it started appearing in the menu as soon as you go to a new page.

In 19.07.3, the new app doesn't appear in the menu even after changing page. A logout and login is needed.

I tried the bootstrap theme and the following apps:

- luci-app-openvpn
- luci-app-nlbwmon

To reproduce:

- log in
- go to System / Software
- update lists
- install `luci-app-nlbwmon`
- go to (for instance) Status / Overview
- the "Bandwidth Monitor" menu still does not appear
- log out / log in : the menu now appears as expected

**Comment by jow-** [comment]
Yep, its a known issue which we cannot fix for 19.07.3 anymore. I added a note to the release notes. A fix will entail an update to luci-app-opkg which will flush the SessionStorage menu cache after a package install operation.

**Comment by singalen** [comment]
Is there a workaround? I wouldn't like to reboot.
I tried 
```
/etc/init.d/uhttpd stop
rm -rf /tmp/luci-*
/etc/init.d/uhttpd start
```
but it didn't help.

**Comment by zorun** [comment]
Would it be enough to backport the (secondary) fix from dc57e4bc6c83d6c869c318256057e3636bfc5d66 ?

**Comment by jow-** [comment]
@zorun - yes, most likely. Not entirely sure if we'd need further backports in the dispatcher code wrt. server side cache invalidation

**Comment by zorun** [comment]
It turns out that simply backporting the fix is enough, see #4402

**Comment by zorun** [comment]
The fix is merged, thanks jow!
